Game/Resources/Constant Scene/Minecart/Script/Camera.as

```diff
@@ -28,9 +28,11 @@ class Camera{
     
     void Update(float deltaTime){
         // Only control camera with mouse if we're not running in VR.
-        if (!IsVRActive())
-            MouseUpdate();
-            
+        if (!IsVRActive()){
+            //MouseUpdate();
+            self.RotateYaw(0.02f);
+            self.position.y = -2;
```

Comment:
Why do we force the camera to rotate in place if we are playing without VR?

Reply:
For the scripted test. simulate camera movment